creator/studies/data_generator/ingest_package/extract_configs/biospec.py

```diff
@@ -0,0 +1,81 @@
+"""
```

Comment:
Do you mean add this biospecimen extract config to that app? Would we be able to run it locally?

Reply:
The extract configs for the study generator are specific to the data files that the study generator creates. I suppose if the study generator were to create data files that match our document types then it would make sense to reuse the extract configs inside that app. But right now it just creates the bare minimum tabular files needed to create a study.

Reply:
I believe the extract configs should be pretty similar. You would have to render them first before using them, but I would expect them to be pretty identical. I suppose because these are specific to the dataservice and only intended to be run on generated data, it will be ok.